Makefile

```diff
@@ -7,7 +7,13 @@ XYZ = node_modules/.bin/xyz --repo git@github.com:sanctuary-js/sanctuary.git --s
 
 
 .PHONY: all
-all: README.md
+all: LICENSE README.md
+
+.PHONY: LICENSE
+LICENSE:
+	cp -- '$@' '$@.orig'
+	sed 's/Copyright (c) .* Sanctuary/Copyright (c) $(shell git log --date=format:%Y --pretty=format:%ad | sort -r | head -n 1) Sanctuary/' '$@.orig' >'$@'
```

Comment:
You can remove the need for `sort` and `head` if you just want the year of the latest commit by passing `-1` as an argument to `git log`.
```
git log -1 --date=format:%Y --pretty=format:%ad
```